src/components/HappyThought.jsx

```diff
@@ -0,0 +1,33 @@
+import { useState } from 'react'
+
+export const HappyThought = ({ message, id }) => {
+const [likeMessage, setLikeMessage] = useState(0)
+
+// POST request and a counter for liking other users' messages
+const handleLike = () => {
+    fetch(`https://happy-thoughts-ux7hkzgmwa-uc.a.run.app/thoughts/${id}/like`, {
```

Comment:
nice work, did you console log the id? 

I played around with the deployed site I could see that the hearts added when I clicked, but if you are to refresh the page, the hearts I've just added are not counted. So something is amiss here? is it possible the props are not passed correctly? perhaps they don't need {} around them.